src/Backups/BackupEntriesCollector.cpp

```diff
@@ -397,7 +397,7 @@ void BackupEntriesCollector::gatherDatabasesMetadata()
 
             case ASTBackupQuery::ElementType::ALL:
             {
-                for (const auto & [database_name, database] : DatabaseCatalog::instance().getDatabases())
+                for (const auto & [database_name, database] : DatabaseCatalog::instance().getDatabases(false))
```

Comment:
We don't have BACKUP for catalogs, so I guess it is safe to pass `false`.

Reply:
But we should backup catalogs definition itself?